relay-metrics/src/protocol.rs

```diff
@@ -531,12 +552,12 @@ mod tests {
 
     #[test]
     fn test_parse_counter() {
-        let s = "foo:42|c";
+        let s = "transactions/foo:42|c";
```

Comment:
Let's leave these tests, this is still a valid payload to retain statsd compatibility.